WordPress/src/main/java/org/wordpress/android/models/Capability.java

```diff
@@ -0,0 +1,22 @@
+package org.wordpress.android.models;
+
+/**
+ * Used to decide what can the current user do in a particular blog
+ * A list of capabilities can be found in: https://codex.wordpress.org/Roles_and_Capabilities#Capabilities
+ */
+public enum Capability {
+    EDIT_USERS("edit_users"), // Check if user can change another user's role
+    LIST_USERS("list_users"), // Check if user can visit People page
+    PROMOTE_USERS("promote_users"); // Check if user can invite another user
+
+    private final String label;
+
+    Capability(String label) {
+        this.label = label;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
I disagree with this. If you look at how we use `toString` in other enums, you'll see that we do exactly the same thing. I think this is the way to use it as general purpose, not implementing and just using the "LIST_USERS" as a string is not, considering we're never going to care about the raw values.

You can check `PostStatus`, `CommentStatus`, `NotificationsSettings.Type` as examples. Cheers!